Content.Client/Launcher/LauncherConnectingGui.xaml.cs

```diff
@@ -51,6 +67,12 @@ public LauncherConnectingGui(LauncherConnecting state)
             var edim = IoCManager.Resolve<ExtendedDisconnectInformationManager>();
             edim.LastNetDisconnectedArgsChanged += LastNetDisconnectedArgsChanged;
             LastNetDisconnectedArgsChanged(edim.LastNetDisconnectedArgs);
+            _cfg.OnValueChanged(CCVars.LoginTipsDataset, SetLoginTipsDataset, true);
+        }
```

Comment:
Events need to be unsubscribed or they leak, especially in a UI control.

Reply:
okay i use UnsubValueChanged